zebROS_ws/src/tf_object_detection/src/test_detection.py

```diff
@@ -1,230 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Can we also leave this file in there?  I could imagine using it on laptops or desktops for testing if they have a gpu but not jetson-utils installed

Reply:
Also need to restore this file